auth/discord.php

```diff
@@ -214,7 +214,7 @@ function assign_user_good_standing($user_discord_id) {
 										</div>
 										<?php
 									} catch (Exception $ex) {
-										AdminBot::send_message("[AUTHDIS] Failed to assign user ({$userinfo['id']}) to the Good Standing role.\n{$ex}");
+										AdminBot::send_message("[AUTHDIS] Failed to assign user (http://untro.bo/admin/check-good-standing?u={$userinfo['id']}) to the Good Standing role.\n{$ex}");

```

Comment:
Can we get their user info  in this message as well

Reply:
What user info would you like to be included?